api/audit/signals.py

```diff
@@ -27,9 +26,15 @@ def call_webhooks(sender, instance, **kwargs):
         if instance.project
         else instance.environment.project.organisation
     )
-    call_organisation_webhooks.delay(
-        args=(organisation.id, data, WebhookEventType.AUDIT_LOG_CREATED.value)
-    )
+
+    if (
+        not settings.DISABLE_WEBHOOKS
+        and organisation.webhooks.filter(enabled=True).exists()
+    ):
```

Comment:
Very minor nit: This would look better if it were inverted and used as an early return from the function. Seems weird to wait to the bottom of the function to discover that `DISABLE_WEBHOOKS` was set.